CHANGELOG-Agoric.md

```diff
@@ -36,10 +36,26 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
 
-## [v0.34.23-alpha.agoric.2]
+## [v0.34.27-alpha.agoric.3]
+
+* Merge `agoric-labs/tendermint` v0.34.23-alpha.agoric.4.
+
+## [v0.34.23-alpha.agoric.4]
+
+* Lower default `BlockParams.MaxBytes` to 5MB to mitigate asa-2023-002 
+
```

Comment:
Drop this since we won't be adding this tag to this repository, but move this description up to line 41.

Reply:
This is a merge of the tendermint commit. We should only resolve merge conflicts, not introduce content changes. Is there a reason to exclude this version from the log when other changes that agoric made are included, associated to the respective version where the change was originally made?

Regarding ordering, they're purposely not in "version order" to make clear that `v0.34.27-alpha.agoric.2` doesn't include this change.

Reply:
This is a bit of a gray area, so proceed if you feel strongly, but I'd rather see this as its own commit since we're no longer lifting tendermint changes into cometbft.

Reply:
In general I prefer to merge instead of cherry-picking or porting, as it's clearer what is and isn't integrated.